core/config/project_settings.cpp

```diff
@@ -1328,6 +1328,7 @@ ProjectSettings::ProjectSettings() {
 	GLOBAL_DEF_BASIC(PropertyInfo(Variant::STRING, "display/window/stretch/mode", PROPERTY_HINT_ENUM, "disabled,canvas_items,viewport"), "disabled");
 	GLOBAL_DEF_BASIC(PropertyInfo(Variant::STRING, "display/window/stretch/aspect", PROPERTY_HINT_ENUM, "ignore,keep,keep_width,keep_height,expand"), "keep");
 	GLOBAL_DEF_BASIC(PropertyInfo(Variant::FLOAT, "display/window/stretch/scale", PROPERTY_HINT_RANGE, "0.5,8.0,0.01"), 1.0);
+	GLOBAL_DEF_BASIC(PropertyInfo(Variant::STRING, "display/window/stretch/stretch", PROPERTY_HINT_ENUM, "fractional,integer"), "fractional");
```

Comment:
Are we going to call this a stretch/stretch property? Isn't there a better name?

Reply:
The added description calls it a "policy" which is a bit less generic than "scale mode", but it's also less self-explanatory if you are not a native speaker, perhaps?

@Riteo What's your take as the creator of this feature? :)